cmd/src/batch_repositories.go

```diff
@@ -90,31 +91,34 @@ Examples:
 			return err
 		}
 
-		repoCount := 0
-		for _, on := range spec.On {
-			repos, _, err := svc.ResolveRepositoriesOn(ctx, &on)
-			if err != nil {
-				return errors.Wrapf(err, "Resolving %q", on.String())
+		_, repos, err := svc.ResolveWorkspacesForBatchSpec(ctx, spec, allowUnsupported, allowIgnored)
```

Comment:
As discussed, this doesn't show the individual queries anymore. A follow-up will deprecate this command and create a new workspaces command as a replacement, that has a more elaborate output, and also properly supports workspaces as a native feature.